recipes/celeries/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "celeries" %}
+{% set version = "0.1.0" %}
+{% set python_min = "3.8" %}
```

Comment:
Only set `python_min` if the minimum required version is higher than the conda-forge default. 


```suggestion
```